### PR TITLE
SQL script consolidation

### PR DESCRIPTION
**Pull request by labkey-ians**

### CODE REVIEW

File: assay/resources/schemas/dbscripts/postgresql/assay-19.20-19.30.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-- Repackage AssayDesignerRole from study to assay
```
Comment (labkey-adam) [inline assay/resources/schemas/dbscripts/postgresql/assay-19.20-19.30.sql]: Delete - this script will never run

File: assay/resources/schemas/dbscripts/sqlserver/assay-19.20-19.30.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+-- Repackage AssayDesignerRole from study to assay
```
Comment (labkey-adam) [inline assay/resources/schemas/dbscripts/sqlserver/assay-19.20-19.30.sql]: Delete - this script will never run

**Comment by labkey-jeckels** [comment]
This is a late-breaking request, I know, but I think we'll want a mothership schema change for https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=38803

If we can hold on that module it would be great, otherwise I'll deal with it as a 19.31 update.

**Comment by labkey-adam** [comment]
Ian will back out the mothership version bump. We're assuming we'll deal with the new script and version bump on the 19.3 branch.

**Comment by labkey-jeckels** [comment]
@labkey-adam yes, exactly. Thank you both

**Pull request merged**